test/t8_geometry/t8_test_point_inside.cxx

```diff
@@ -33,161 +33,208 @@
 #include <t8_cmesh/t8_cmesh_geometry.h>
 #include <t8_geometry/t8_geometry_implementations/t8_geometry_linear.hxx>
 
-/* This function creates a single element of the specified element class.
- * It the creates a bunch of points, some of which lie whithin the element, some
+/* This function creates uniform forests of the specified element class.
+ * It then iterates over the forest's elements and 
+ * creates a bunch of points, some of which lie whithin the element, some
  * not. For each point we call t8_forest_element_point_inside and check whether
- * it gives the correct result. */
-/*
- * TODO: - Use more than one refinement level.
- *       - Does the new barycentric coordinate test work with HEX and PRISM?
+ * it gives the correct result.
+ * 
+ * \param [in] comm     MPI communicator
+ * \param [in] eclass   Element class to check
+ * \param [in] maxlevel Up to this level uniform forests are created.
  */
 static void
-t8_test_point_inside_level0 (sc_MPI_Comm comm, t8_eclass_t eclass)
+t8_test_point_inside (sc_MPI_Comm comm, t8_eclass_t eclass, int maxlevel)
```

Comment:
Shouldn't we convert this to a gtest?

Reply:
Yes we should. I did not have the time, though.